packages/eslint-plugin/src/rules/shorthand-property-sorting/utils.ts

```diff
@@ -0,0 +1,68 @@
+export const shorthandFor: Record<string, number> = {
```

Comment:
replace this with `shorthandBuckets` from `packages/utils/src/shorthand.ts`

https://github.com/atlassian-labs/compiled/blob/4ed15ebec15830eb3b67069a9f8f54986f211594/packages/utils/src/shorthand.ts

(you'll need to rebase from master)